roles/confluent.kafka_connect_replicator/tasks/main.yml

```diff
@@ -0,0 +1,307 @@
+---
+- include_role:
+    name: confluent.common
+  when: not common_role_completed|bool
+
+- name: Gather OS Facts
+  setup:
+    # Only gathers items in list, filters out the rest
+    filter: "{{item}}"
+    gather_subset:
+      - '!all'
+  loop:
+    - ansible_os_family
+    - ansible_fqdn
+
+# Install Packages
+- name: Install the Kafka Connect Replicator Packages
+  yum:
+    name: "{{item}}{{confluent_package_redhat_suffix}}"
+    state: latest
+  loop: "{{kafka_connect_replicator_packages}}"
+  when:
+    - ansible_os_family == "RedHat"
+    - installation_method == "package"
+  tags: package
+
+- name: Install the Kafka Connect Replicator Packages
+  apt:
+    name: "{{item}}{{confluent_package_debian_suffix}}"
+  loop: "{{kafka_connect_replicator_packages}}"
+  when:
+    - ansible_os_family == "Debian"
+    - installation_method == "package"
+  tags: package
+
+# Configure environment
+- name: Create Kafka Connect Replicator Group
+  group:
+    name: "{{kafka_connect_replicator_group}}"
+
+- name: Create Kafka Connect Replicator User
+  user:
+    name: "{{kafka_connect_replicator_user}}"
+    comment: "Confluent Replicator User"
+    system: true
+    group: "{{kafka_connect_replicator_group}}"
+
+- name: Configure TLS for Kafka Connect Replicator Host
+  include_role:
+    name: confluent.ssl
+  vars:
+    truststore_storepass: "{{kafka_connect_replicator_truststore_storepass}}"
+    truststore_path: "{{kafka_connect_replicator_truststore_path}}"
+    keystore_path: "{{kafka_connect_replicator_keystore_path}}"
+    keystore_storepass: "{{kafka_connect_replicator_keystore_storepass}}"
+    keystore_keypass: "{{kafka_connect_replicator_keystore_keypass}}"
+    service_name: confluent-replicator
+    user: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+    hostnames: "{{ [inventory_hostname, ansible_fqdn, ansible_host, ansible_ssh_host] | unique }}"
+    ca_cert_path: "{{kafka_connect_replicator_ca_cert_path}}"
+    cert_path: "{{kafka_connect_replicator_cert_path}}"
+    key_path: "{{kafka_connect_replicator_key_path}}"
+    export_certs: "{{kafka_connect_replicator_export_certs}}"
+    ssl_ca_cert_file_path: "{{kafka_connect_replicator_ssl_ca_cert_path}}"
+    ssl_signed_cert_filepath: "{{kafka_connect_replicator_ssl_cert_path}}"
+    ssl_key_filepath: "{{kafka_connect_replicator_ssl_key_path}}"
+    ssl_key_password: "{{kafka_connect_replicator_ssl_key_password}}"
+    ssl_truststore_file_path: "{{kafka_connect_replicator_ssl_truststore_file_path}}"
+    ssl_keystore_file_path: "{{kafka_connect_replicator_ssl_keystore_file_path}}"
+  when: kafka_connect_replicator_ssl_enabled|bool or kafka_connect_replicator_ssl_provided_keystore_and_truststore|bool
```

Comment:
For other components- `<component>_ssl_enabled: true` means that component's rest endpoint has tls on. The way its written right now is you allow for certs per kafka connection. Are we allowing for a unique set of certs for the Rest endpoint TLS as well?

Reply:
No we are not, as we currently do not allow this on Kafka clusters either.  I think it's net new work, that we should implement across the platform in a future release.